both/collections/round.js

```diff
@@ -30,6 +30,13 @@ _.extend(Round.prototype, {
     assert(this.eventCode);
     return wca.eventByCode[this.eventCode].solveTimeFields;
   },
+  roundCode: function() {
+    if(this.isLast()) {
+      return this.softCutoff ? 'c' : 'f';
+    } else {
+      return (this.softCutoff ? 'degc' : '123f')[this.nthRound-1];
```

Comment:
I was thinking that `wca.js` is a place to put numbers that come from the regulations, and logic required to produce the [WCA competition JSON format](https://github.com/cubing/worldcubeassociation.org/wiki/WCA-Competition-JSON-Format).

After thinking about this for longer, I would rather that the CCM codebase not deal with WCA round codes unless it is importing or exporting data. That's cleanup for another PR, though (I've created https://github.com/cubing/ccm/issues/247 to track this). Lets leave this code alone for now.